task-json/us.js

```diff
@@ -0,0 +1,1130 @@
+    "Caldwell",
+    "Coeur d'Alene",
+    "Twin Falls"
+  ],
+  "Alabama": [
+    "Birmingham",
+    "Montgomery",
+    "Mobile",
+    "Huntsville",
+    "Tuscaloosa",
+    "Hoover",
+    "Dothan",
+    "Auburn",
+    "Decatur",
+    "Madison",
+    "Florence",
+    "Gadsden"
+  ],
+  "Iowa": [
+    "Des Moines",
+    "Cedar Rapids",
+    "Davenport",
+    "Sioux City",
+    "Iowa City",
+    "Waterloo",
+    "Council Bluffs",
+    "Ames",
+    "West Des Moines",
+    "Dubuque",
+    "Ankeny",
+    "Urbandale",
+    "Cedar Falls"
+  ],
+  "Arkansas": [
+    "Little Rock",
+    "Fort Smith",
+    "Fayetteville",
+    "Springdale",
+    "Jonesboro",
+    "North Little Rock",
+    "Conway",
+    "Rogers",
+    "Pine Bluff",
+    "Bentonville"
+  ],
+  "Utah": [
+    "Salt Lake City",
+    "West Valley City",
+    "Provo",
+    "West Jordan",
+    "Orem",
+    "Sandy",
+    "Ogden",
+    "St. George",
+    "Layton",
+    "Taylorsville",
+    "South Jordan",
+    "Lehi",
+    "Logan",
+    "Murray",
+    "Draper",
+    "Bountiful",
+    "Riverton",
+    "Roy"
+  ],
+  "Rhode Island": [
+    "Providence",
+    "Warwick",
+    "Cranston",
+    "Pawtucket",
+    "East Providence",
+    "Woonsocket"
+  ],
+  "Mississippi": [
+    "Jackson",
+    "Gulfport",
+    "Southaven",
+    "Hattiesburg",
+    "Biloxi",
+    "Meridian"
+  ],
+  "South Dakota": [
+    "Sioux Falls",
+    "Rapid City"
+  ],
+  "Connecticut": [
+    "Bridgeport",
+    "New Haven",
+    "Stamford",
+    "Hartford",
+    "Waterbury",
+    "Norwalk",
+    "Danbury",
+    "New Britain",
+    "Meriden",
+    "Bristol",
+    "West Haven",
+    "Milford",
+    "Middletown",
+    "Norwich",
+    "Shelton"
+  ],
+  "South Carolina": [
+    "Columbia",
+    "Charleston",
+    "North Charleston",
+    "Mount Pleasant",
+    "Rock Hill",
+    "Greenville",
+    "Summerville",
+    "Sumter",
+    "Goose Creek",
+    "Hilton Head Island",
+    "Florence",
+    "Spartanburg"
+  ],
+  "New Hampshire": [
+    "Manchester",
+    "Nashua",
+    "Concord"
+  ],
+  "North Dakota": [
+    "Fargo",
+    "Bismarck",
+    "Grand Forks",
+    "Minot"
+  ],
+  "Montana": [
+    "Billings",
+    "Missoula",
+    "Great Falls",
+    "Bozeman"
+  ],
+  "Delaware": [
+    "Wilmington",
+    "Dover"
+  ],
+  "Maine": [
+    "Portland"
+  ],
+  "Wyoming": [
+    "Cheyenne",
+    "Casper"
+  ],
+  "West Virginia": [
+    "Charleston",
+    "Huntington"
+  ],
+  "Vermont": [
+    "Burlington"
+  ]
+};
+function list_state(){
+    const list_state=[];
+    for (var key in states) {
+        if (states.hasOwnProperty(key)) {
+          list_state.push(key);
+        }
+    }
+    return list_state;
+  }
+  const list_states= list_state();
+function number_of_city (name)
+{
+    if(typeof name === 'string' || name instanceof String){
+        for (var key in states ){
+            if(key===name){
+                var count =Object.keys(states[name]).length;
+               return count;
+            }
+        }

```

Comment:
think in a way to make it shorter
another point, don't create a variable if you want to return it directly (no processing)
`var count =Object.keys(states[name]).length;
               return count;`